diff -urN mknbi-1.2.org/Makefile mknbi-1.2/Makefile
--- mknbi-1.2.org/Makefile	Wed May  2 11:41:04 2001
+++ mknbi-1.2/Makefile	Wed May  2 11:41:33 2001
@@ -76,17 +76,17 @@
 
 # New 32-bit first stage setup program
 first32.linux:	start32.o first32.o printf.o
-	$(LD) -N -Ttext $(FIRSRELOC) -e _start -oformat binary -o $@ start32.o first32.o printf.o
+	$(LD) -N -Ttext $(FIRSRELOC) -e _start --oformat binary -o $@ start32.o first32.o printf.o
 	@if [ `wc -c < $@` -gt 4096 ]; then echo Binary too large; fi
 
 # New 32-bit first stage protected mode call setup program
 first32pm.linux:	start32pm.o first32pm.o printf.o
-	$(LD) -N -Ttext $(FIRSRELOC) -e _start -oformat binary -o $@ start32pm.o first32pm.o printf.o
+	$(LD) -N -Ttext $(FIRSRELOC) -e _start --oformat binary -o $@ start32pm.o first32pm.o printf.o
 	@if [ `wc -c < $@` -gt 4096 ]; then echo Binary too large; fi
 
 # New 32-bit first stage ELF setup program
 first32elf.linux:	start32pm.o first32elf.o printf.o
-	$(LD) -N -Ttext $(FIRSRELOC) -e _start -oformat binary -o $@ start32pm.o first32elf.o printf.o
+	$(LD) -N -Ttext $(FIRSRELOC) -e _start --oformat binary -o $@ start32pm.o first32elf.o printf.o
 	@if [ `wc -c < $@` -gt 4096 ]; then echo Binary too large; fi
 
 start32.o:	start32.S
@@ -135,11 +135,11 @@
 
 # Menu first stage program
 menu:	startmenu.o menu.o string.o printf.o ansiesc.o
-	$(LD) -N -Ttext $(MENURELOC) -e _start -oformat binary -o $@ startmenu.o menu.o string.o printf.o ansiesc.o
+	$(LD) -N -Ttext $(MENURELOC) -e _start --oformat binary -o $@ startmenu.o menu.o string.o printf.o ansiesc.o
 
 # Another menu program, this one simpler
 menu-simple:	startmenu.o menu-simple.o string.o printf.o ansiesc.o
-	$(LD) -N -Ttext $(MENURELOC) -e _start -oformat binary -o $@ startmenu.o menu-simple.o string.o printf.o ansiesc.o
+	$(LD) -N -Ttext $(MENURELOC) -e _start --oformat binary -o $@ startmenu.o menu-simple.o string.o printf.o ansiesc.o
 
 startmenu.o:	startmenu.S
 	gcc -E -DRELOC=$(MENURELOC) $(OLDGAS) startmenu.S | $(AS) -o startmenu.o
